src/Controller/SiteController.php

```diff
@@ -24,4 +31,66 @@ public function index(ServerRequestInterface $request): ResponseInterface
             ]
         );
     }
+
+    public function about(ServerRequestInterface $request): ResponseInterface
+    {
+        return $this->render(
+            'about',
+            [
+                'csrf' => $request->getAttribute(Csrf::REQUEST_NAME)
+            ]
+        );
+    }
+
+    public function contact(
+        ApplicationParameters $app,
```

Comment:
Not used.

Reply:
@samdark widgets I have not uploaded the modification because I hope to finish nav today, to use yii-bulma.

Reply:
yii-bulma is there.